backend/apps/ifc_validation/admin.py

```diff
@@ -129,6 +238,11 @@ def duration_text(self, obj):
         else:
             return None
 
+    def has_add_permission(self, request):
+
+        # disable add via Admin ('+ Add' button)
+        return False
```

Comment:
Maybe in the future isolate this into a class and inherit from? `class ValidationTaskAdmin(BaseAdmin, NonAdminAddable):`

Reply:
Done - thanks!